connect.go

```diff
@@ -217,6 +219,13 @@ func (opts Options) connect() (*Conn, error) {
 		opts.MaxReconnect = 0
 	}
 
+	if opts.ConnectionToken != "" && opts.Password != "" {
+		return nil, memphisError(errors.New("Can't connect with connection token and password - must choose one method"))
```

Comment:
You have to connect with only one of the following methods: connection token / password